job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/RundeckContext.groovy

```diff
@@ -36,4 +37,21 @@ class RundeckContext implements Context {
     void shouldFailTheBuild(boolean shouldFailTheBuild = true) {
         this.shouldFailTheBuild = shouldFailTheBuild
     }
+
+    //Requires Rundeck plugin 3.4+
```

Comment:
First I tried (before changing it into a comment), but I've got:
```
startup failed:
       @RequiresPlugin(id = 'rundeck', minimumVersion = '3.4')
       ^
/.../job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/RundeckContext.groovy: 42: no jobManagement field in javaposse.jobdsl.dsl.helpers.publisher.RundeckContext
```
Can that annotation be used with a method like that?